File: Assignment 3/C# version/Exercise 6/PointMain.cs
using System;
using Assignment1;

class PointMain
{
	public static void Main(string[] args)
	{
		Point p1 = new Point();
		Point p2 = new Point(3, 4);
		Console.WriteLine(p1.toString());
		Console.WriteLine(p2.toString());
		if (p1.isEqualTo(p2))
			Console.WriteLine("The two points are equal");
		double dist = p1.distanceTo(p2);
		Console.WriteLine("Point Distance: " + dist);
		p2.move(5, -2);
		p1.moveToXY(8, 2);
		if (p1.isEqualTo(p2))
			Console.WriteLine("The two points are equal");
	}
}

File: sb223ce_assign3/PointMain.java
package sb223ce_assign3;

public class PointMain {

	public static void main(String[] args) {

		// call the constructor

		Point p1 = new Point();
		Point p2 = new Point(3, 4);

		// call the methods
		System.out.println(p1.toString());
		System.out.println(p2.toString());

		if (p1.isEqualTo(p2))
			System.out.println("The two points are equal");

		double dist = p1.distanceTo(p2);
		System.out.println("Point Distance: " + dist);

		p2.move(5, -2);
		p1.moveToXY(8, 2);

		if (p1.isEqualTo(p2))
			System.out.println("The two points are equal");

	}

}
